pipeline/cactus_workflow.py

```diff
@@ -667,35 +747,35 @@ def run(self):
 ############################################################
 ############################################################
     
-class CactusNormalPhase(CactusPhasesTarget):
+class CactusNormalPhase(CactusPhasesJob):
```

Comment:
Maybe too late :), but don't worry about doing this phase, at least right now. I don't think the normalization phase has been run in years.